rosdep/base.yaml

```diff
@@ -4617,6 +4617,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nlohmann-json:
+  debian: [nlohmann-json-dev]
+  fedra: [json]
```

Comment:
This should be `fedora`.  Also, while the link you pasted *does* show this as the nlohmann version of json, I actually don't think that it is the same one.  If you look at https://apps.fedoraproject.org/packages/json , it seems to be a json-rpc package from glib.  Since I don't see another equivalent package for Fedora, let's just remove it here.

Reply:
good catch

Reply:
Thanks! I fixed in https://github.com/ros/rosdistro/pull/24432/commits/600a51b70b054f97ce633e5a1839a427e504a094